tests/rpc/rpc_03-chain_test.go

```diff
@@ -249,7 +251,8 @@ func TestChainSubscriptionRPC(t *testing.T) { //nolint:tparallel
 		// Check block numbers grow by zero or one in order of responses.
 		for i, blockNumber := range blockNumbers {
 			if i == 0 {
-				assert.Equal(t, uint(1), blockNumber)
+				// the first finalized block could be block 0 as well
+				assert.GreaterOrEqual(t, blockNumber, uint(0))
```

Comment:
this is unneeded since blockNumber is an uint

```suggestion
```

or could we change this to check it's `0` or `1` only?